onnx_tf/handlers/backend/expand.py

```diff
@@ -11,5 +11,12 @@ class Expand(BackendHandler):
   def version_8(cls, node, **kwargs):
     tensor_dict = kwargs["tensor_dict"]
     x, shape = tensor_dict[node.inputs[0]], tensor_dict[node.inputs[1]]
-    ones = tf.ones(shape, dtype=x.dtype)
-    return [x * ones]
+
+    # tf.matmul does not support bool therefore use int32
```

Comment:
Why is tf.matmul relevant here?

Reply:
Yeah, wrong comment, it is tf.math.multiply, not matmul.